src/LibraryManager/LibraryInstallationResult.cs

```diff
@@ -67,5 +72,11 @@ public static LibraryInstallationResult FromCancelled(ILibraryInstallationState
                 Cancelled = true
             };
         }
+
+        /// <summary>Internal use only</summary>
+        public static LibraryInstallationResult FromError(IError[] errors)
```

Comment:
Nit: I'd call it FromErrors (plural) since it takes an array. Also I'd make argument type IEnumerable<IError> so make it more general (and you can still pass in an array).

Reply:
Fixed